src/Nethermind/Nethermind.Merge.Plugin/Data/GetPayloadV2Result.cs

```diff
@@ -3,20 +3,26 @@
 
 using Nethermind.Core;
 using Nethermind.Int256;
+using Newtonsoft.Json;
 
 namespace Nethermind.Merge.Plugin.Data;
 
 public class GetPayloadV2Result
 {
-    public GetPayloadV2Result(Block block, UInt256 blockFees)
+    public GetPayloadV2Result(Block block, UInt256 blockFees, BlobsBundleV1? blobsBundle = null)
```

Comment:
GetPayloadV3Result?

Reply:
Works as V2 when no bundles. We have something similar for ExecutionPayload